src/Core/src/Platform/Android/MauiSwipeView.cs

```diff
@@ -1344,11 +1344,12 @@ void ProgrammaticallyOpenSwipeItem(OpenSwipeItem openSwipeItem, bool animated)
 
 			if (swipeItems == null || swipeItems.Count == 0)
 				return;
+			
+			UpdateSwipeItems();
```

Comment:
With the changes, we measure the SwipeItemViews before getting the threshold. What happened was that the size obtained from each SwipeItemView was zero and the default thjreshold value was used.